src/Pages/seyongyun/Login/Login.scss

```
@@ -0,0 +1,50 @@
+.login {
+  display: flex;
+  justify-content: center;
+  .new {
+    box-sizing: border-box;
+  }
```

```suggestion
.login {
  display: flex;
  justify-content: center;
  
  .new {
    box-sizing: border-box;
  }
```
각 클래스의 스타일 사이에 간격이 있으면 더 좋을 것 같습니다 :)